### PR TITLE
feat: add casper 3 support for gs

### PR DESCRIPTION
**Pull request by niksrc**
This PR adds support for gameserver nodes in `casper-3`. We already had support for handling multiple node labels while building `labelSelector` in `GetNodes`. The only thing left was to make sure that `LabelValues` env variable is properly formatted for use as a [set based label requirement](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement)

### CODE REVIEW

File: internal/config/config_test.go [hunk 1]
```diff
@@ -25,7 +26,7 @@ func TestFromEnv(t *testing.T) {
 	setenv(t, "INTERVAL", "61")
 	setenv(t, "PROVIDER", "digitalocean")
 	setenv(t, "LABEL_KEY", "doks.digitalocean.com/node-pool")
-	setenv(t, "LABEL_VALUES", "sfu")
+	setenv(t, "LABEL_VALUES", "sfu,")
```
Comment (hwoarang) [inline internal/config/config_test.go:29]: Why is this needed?

**Review by hwoarang — COMMENTED** [review]
Overall looks sensible to me. Quite minimal approach :) Would it make sense to change the base branch of the PR to be the `develop` branch so if we merge this we only deploy it in develop so we can make sure nothing brakes? 🤔

File: internal/config/config_test.go (hunk 1 above)
Comment (niksrc) [reply internal/config/config_test.go]: Not really needed here, as I am already testing this in the table-driven tests. Thanks for pointing this out 👍🏼

**Review by axilleastr — APPROVED** [review]

**Review by hwoarang — APPROVED** [review]
